docs/modules/ROOT/pages/partials/con_hermetic_troubleshooting.adoc

```diff
@@ -0,0 +1,12 @@
+.Troubleshooting
+
+ifdef::troubleshooting_builds[]
+
+If your build fails, be sure to look at your logs. Here are a few ways to find them:
+
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check, then from the resulting *Pipeline run details* view, select the *Logs* tab.
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check. In the resulting *Pipeline run details* view, click *build-container*. When the right panel opens, select the *Logs* tab to see a partial view of the log for that build.
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check, then from the resulting *Pipeline run details* view, scroll until you see *Vulnerabilities scan*. Under that, click *View logs*.
+* In Git, go to the pull request for your build, then select the *Checks* tab.
```

Comment:
Checks in GH won't get the full details, and can't link them effectively to the right spot in the UI. It's a reasonable spot to look to verify if a task completed or not, but troubleshooting or digging deeper is best done today in Konflux until some more feature work can be prioritized and completed.

Reply:
@mattreid thanks! i included this bc someone suggested i include it. (can't remember who!) would you like me to delete this or add clarification?

Reply:
To me this line fits better with the theme of verify, than find logs, since the logs aren't accessible in GH. To verify it worked without going to Konflux, they can look at the Checks tab, and see the status of prefetch-dependencies and build-container. The links we're able to embed there just go generically to Konflux though, not to the relevant pipeline details page in Konflux, so it's doubly not helpful, no logs directly in GH, and no direct link to Konflux details which do have the logs.

Reply:
i understand. i think, to avoid any further confusion, i'll just delete it and not add it to the verification section either. this ok? @mattreid